modules/openapi-generator/src/main/resources/typescript-angular/api.service.mustache

```diff
@@ -72,7 +72,7 @@ export class {{classname}} {
         }
         if (typeof this.configuration.basePath !== 'string') {
             if (Array.isArray(basePath) && basePath.length > 0) {
-                basePath = basePath[0];
+                basePath = basePath[0]!;
```

Comment:
How about explicitly checking that this is not nu instead?

Reply:
I added an variable and checked it's value.

I'm not sure about the use of multiple base paths, but in this case the value is checked for falsyness. I could also check if the value is undefined or do an Array.find for the base paths array.